components/web/components/org.wso2.carbon.apimgt.publisher.commons.ui/src/main/fragments/api-access-control/public/css/permissions.css

```diff
@@ -0,0 +1,13 @@
+input[type="checkbox"] {
```

Comment:
Do not use default element to style. use ID or classes

Reply:
Thanks for the review. This file is no longer available in this repository.